frontend/src/container/MetricsApplication/TopOperationsTable.tsx

```diff
@@ -93,7 +93,7 @@ function TopOperationsTable(props: TopOperationsTableProps): JSX.Element {
 			sorter: (a: TopOperationList, b: TopOperationList): number =>
 				a.errorCount - b.errorCount,
 			render: (value: number, record: TopOperationList): string =>
```

Comment:
can you please update the type also would be good

Reply:
yes, sorry missed those. Will add them right away.

Reply:
@palashgdev Sorry for being late.

I had a question, Should I change the `TopOperationList` type itself or the `render` fn types are enough? 

I wanna do it in `TopOperationList` lemme know I'll fix it asap. Thanks.

Reply:
Hi @ksorv 
updating `TopOperationList` would make sense here